testsuite/MDAnalysisTests/coordinates/test_gms.py

```diff
@@ -69,47 +65,52 @@ def _calcFD(u):
         z2 = np.sqrt(sum(pp ** 2))
         return z1 - z2
 
-    def test_rewind(self):
-        self.u.trajectory.rewind()
-        assert_equal(self.u.trajectory.ts.frame, 0, "rewinding to frame 0")
+    def test_rewind(self, u):
+        u.trajectory.rewind()
+        assert_equal(u.trajectory.ts.frame, 0, "rewinding to frame 0")
 
-    def test_next(self):
-        self.u.trajectory.rewind()
-        self.u.trajectory.next()
-        assert_equal(self.u.trajectory.ts.frame, 1, "loading frame 1")
+    def test_next(self, u):
+        u.trajectory.rewind()
+        u.trajectory.next()
+        assert_equal(u.trajectory.ts.frame, 1, "loading frame 1")
 
-    def test_dt(self):
-        assert_almost_equal(self.u.trajectory.dt,
+    def test_dt(self, u):
+        assert_almost_equal(u.trajectory.dt,
                             1.0,
                             4,
                             err_msg="wrong timestep dt")
 
-    def test_step5distances(self):
-        assert_almost_equal(self._calcFD(self.u), self.step5d, decimal=5,
+    def test_step5distances(self, u):
+        assert_almost_equal(self._calcFD(u), self.step5d, decimal=5,
                             err_msg="Wrong 1-4 atom distance change after "
-                            "5 steps for {}".format(self.flavour))
+                                    "5 steps for {}".format(self.flavour))
 
 
 class TestGMSReader(_GMSBase):
-    __test__ = True
-    def setUp(self):
-        self.u =  mda.Universe(GMS_ASYMOPT)
-        self.n_frames = 21
-        self.flavour = "GAMESS C1 optimization"
-        self.step5d = -0.0484664
+    n_frames = 21
+    flavour = "GAMESS C1 optimization"
+    step5d = -0.0484664
+
+    @pytest.fixture()
+    def u(self):
+        return mda.Universe(GMS_ASYMOPT)
+
 
 class TestGMSReaderSO(_GMSBase):
-    __test__ = True
-    def setUp(self):
-        self.u = mda.Universe(GMS_SYMOPT)
-        self.n_frames = 8
-        self.flavour = "GAMESS D4H optimization"
-        self.step5d = 0.227637
+    n_frames = 8
+    flavour = "GAMESS D4H optimization"
+    step5d = 0.227637
+
+    @pytest.fixture()
+    def u(self):
+        return mda.Universe(GMS_SYMOPT)
```

Comment:
it makes more sense to define this fixture in the base class and use a common class variable like `self.trajectory_file` to set it up instead of having to write the fixture for every child class. You already did that for other classes in coordinates.